VideoAPI/VideoApi.Services/AudioPlatformService.cs

```diff
@@ -75,6 +86,11 @@ public async Task<AudioPlatformServiceResponse> CreateAudioApplicationAsync(Guid
             }
             catch (AudioPlatformException ex)
             {
+                if (ex.StatusCode == HttpStatusCode.NotFound)
```

Comment:
The service class shouldn't be returning a contract type directly. It overstretches it's responsibility. The controller should be executing the service class and transforming the result into a response

Reply:
The AudioPlatformServiceResponse is a service response class, the controller then decides on the ObjectResult from this.

Reply:
Okay, I was mislead when I saw the model in the contracts projects.